nodes/src/nodes/db_neo4j/neo4j_driver.py

```
@@ -195,8 +311,33 @@ def _tool_validate(self, *, tool_name: str, input_obj: Any) -> None:
             if not question or not isinstance(question, str) or not question.strip():
                 raise ValueError('"question" is required and must be a non-empty string')
 
+        elif name == 'create_node':
+            if not isinstance(input_obj, dict):
+                raise ValueError('Tool input must be a JSON object')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)